crates/biome_css_analyze/src/lint/nursery/no_unknown_property.rs

```diff
@@ -0,0 +1,86 @@
+use biome_analyze::{context::RuleContext, declare_rule, Ast, Rule, RuleDiagnostic};
+use biome_console::markup;
+use biome_css_syntax::CssGenericProperty;
+use biome_rowan::{AstNode, TextRange};
+
+use crate::utils::{is_kown_properties, vendor_prefixed};
+
+declare_rule! {
+    /// Disallow unknown properties.
+    ///
+    /// This rule considers properties defined in the `CSS Specifications and browser specific properties` to be known.
```

Comment:
```suggestion
    /// This rule considers properties defined in the CSS Specifications and browser specific properties to be known.
```

Is the code block needed?